examples/world3/figures/world3.jl

```diff
@@ -88,3 +147,175 @@ system = world3_historicalrun(agriculture_tables=agr_tables_7_13)
+    (dr.cdr,   0, 50,   "cdr"),
+]
+
+
+agr_tables_7_20[:llmy2] = (1.2, 1.0, 0.9, 0.8, 0.75, 0.7, 0.67, 0.64, 0.62, 0.6)
+
+system = world3_historicalrun(nonrenewable_params=nr_parameters_7_20, pollution_params=pol_parameters_7_20, agriculture_params=agr_parameters_7_21, nonrenewable_tables=nr_tables_7_20, agriculture_tables=agr_tables_7_20)
+sol_7_22 = WorldDynamics.solve(system, (1900, 2100))
+
+plotvariables(sol_7_22, (t, 1900, 2100), fig_7_22_variables, name="Fig. 7.22", showlegend=true, showaxis=true, colored=true)
+
+
+pop_parameters_7_34 = World3.Pop4.getparameters()
+pop_parameters_7_34[:zpgt] = 1975
+
+system = world3_historicalrun(pop_params=pop_parameters_7_34)
+sol_7_34 = WorldDynamics.solve(system, (1900, 2100))
+
+plotvariables(sol_7_34, (t, 1900, 2100), fig_7_7_variables, name="Fig. 7.34", showlegend=true, showaxis=true, colored=true)
+
+
+cap_parameters_7_35 = World3.Capital.getparameters()
+cap_parameters_7_35[:alic1] = 21
+cap_parameters_7_35[:alsc2] = 30
+
+system = world3_historicalrun(capital_params=cap_parameters_7_35)
+sol_7_35 = WorldDynamics.solve(system, (1900, 2100))
+
+plotvariables(sol_7_35, (t, 1900, 2100), fig_7_7_variables, name="Fig. 7.35", showlegend=true, showaxis=true, colored=true)
+
+
+cap_tables_7_36 = World3.Capital.gettables()
+cap_tables_7_36[:isopc2] = (60, 450, 960, 1500, 1830, 2175, 2475, 2700, 3000)
+
+agr_tables_7_36 = World3.Agriculture.gettables()
+agr_tables_7_36[:ifpc2] = (345, 720, 1035, 1275, 1455, 1605, 1725, 1815, 1875)
+
+system = world3_historicalrun(capital_tables=cap_tables_7_36, agriculture_tables=agr_tables_7_36)
+sol_7_36 = WorldDynamics.solve(system, (1900, 2100))
+
+plotvariables(sol_7_36, (t, 1900, 2100), fig_7_7_variables, name="Fig. 7.36", showlegend=true, showaxis=true, colored=true)
+
+
+pop_parameters_7_37 = World3.Pop4.getparameters()
+pop_parameters_7_37[:zpgt] = 1975
+
+system = world3_historicalrun(pop_params=pop_parameters_7_37, capital_tables=cap_tables_7_36, agriculture_tables=agr_tables_7_36)
+sol_7_37 = WorldDynamics.solve(system, (1900, 2100))
+
+plotvariables(sol_7_37, (t, 1900, 2100), fig_7_7_variables, name="Fig. 7.37", showlegend=true, showaxis=true, colored=true)
+
+
+cap_parameters_7_38 = World3.Capital.getparameters()
+cap_parameters_7_38[:iet] = 1990
+cap_parameters_7_38[:iopcd] = 320
+cap_parameters_7_38[:alic2] = 21
+cap_parameters_7_38[:alsc2] = 30
+
+nr_parameters_7_38 = World3.NonRenewable.getparameters()
+nr_parameters_7_38[:nruf2] = 0.125
+
+pol_parameters_7_38 = World3.Pollution.getparameters()
+pol_parameters_7_38[:ppgf21] = 0.25
+
+system = world3_historicalrun(pop_params=pop_parameters_7_37, capital_params=cap_parameters_7_38, nonrenewable_params=nr_parameters_7_38, pollution_params=pol_parameters_7_38, capital_tables=cap_tables_7_36, agriculture_tables=agr_tables_7_36)
+sol_7_38 = WorldDynamics.solve(system, (1900, 2100))
+
+plotvariables(sol_7_38, (t, 1900, 2100), fig_7_7_variables, name="Fig. 7.38", showlegend=true, showaxis=true, colored=true)
```

Comment:
It's missing a blank line at the end of the file. But since it was already merged, we can fix this later.